dependencies.gradle

```diff
@@ -17,34 +17,35 @@ ext {
 
     libraries = [
             app : [
-                    autoValue          : 'com.google.auto.value:auto-value:1.5.1',
```

Comment:
Why is the whole block reformatted? You just removed one line as far as I can tell, shouldn't have impacted the rest

Reply:
I'm afraid I have once more the wrong format for spaces... We can check and fix it once and for all